hubtools/src/caboose.rs

```diff
@@ -112,13 +123,33 @@ impl CabooseBuilder {
         self
     }
 
+    pub fn sign<S: Into<String>>(mut self, sign: S) -> Self {
+        self.sign = Some(sign.into());
+        self
+    }
```

Comment:
and this in https://github.com/oxidecomputer/hubtools/commit/943c4bbe6b50d1ab635d085d6204895fb4154e79